src/particles.cpp

```diff
@@ -274,8 +274,7 @@ void ParticleSpawner::step(float dtime, ClientEnvironment* env)
 				// particle if it is attached to an unloaded
 				// object.
 				if (!unloaded) {
-					v3f pos = random_v3f(m_minpos, m_maxpos)
-							+ attached_offset;
+					v3f pos = random_v3f(m_minpos, m_maxpos);
```

Comment:
I would prefer that the pos += attached_offset line was deleted instead, since that would make this branch consistent with the other one. Not a big deal though.